### PR TITLE
Use local hostname resolving to get rid of IP ranges.

### PR DESCRIPTION
**Pull request by Timple**
Lets evaluate this on amigo for some challenges

### CODE REVIEW

File: setup/tue.bash_functions [hunk 1]
```diff
@@ -803,7 +803,7 @@ function tue-robocup-update
             cd $pkg_dir
             local current_url=`git config --get remote.origin.url`
 
-            if echo "$current_url" | grep -q "192.168.44.10"
+            if echo "$current_url" | grep -q "froboticssrv.local"
```
Comment (LoyVanBeek) [inline setup/tue.bash_functions]: typo?

**Comment by LoyVanBeek** [comment]
Who does the DNS resolving now? The router? Do we have the router config back-upped somewhere?

File: setup/tue.bash_functions (hunk 1 above)
Comment (Timple) [reply setup/tue.bash_functions]: Fixed

**Comment by Timple** [comment]
DNS is now done with /etc/hosts file. Router is not involved. So no need for a backup (regarding this PR)

**Pull request closed without merging**